process.md

```diff
@@ -61,19 +61,27 @@ This is for two main reasons:
 2. It's easier/cheaper to make changes before implementation (and you'll feel
    less emotionally invested!)
 
+In general, you should follow the [Tekton Enhancement Proposals
+(`TEP`)](./teps/) process. A Tekton Enhancement Proposal (TEP) is a
+way to propose, communicate and coordinate on new efforts for the
+Tekton project.  You can read the full details of the project in
+[TEP-1](0001-tekton-enhancement-proposal-process.md).
```

Comment:
I think this link should be `./teps/0001-tek...`?

Reply:
ah right :sweat: